src/Commands/config/ConfigCommands.php

```diff
@@ -175,6 +176,85 @@ public function delete($config_name, $options = []) {
   }
 
   /**
+   * Display status of configuration (differences between stored and active configuration).
+   *
+   * @command config-status
+   * @param $label A config directory label (i.e. a key in \$config_directories array in settings.php).
+   * @interact-config-label
```

Comment:
You have to add a @bootstrap annotation like many other commands. Otherwise your command is called unbootstrapped you get the fatal I am seeing now - Fatal error: Call to undefined function config_get_config_directory()